doc/routes.md

```diff
@@ -70,19 +70,34 @@ Complex filters (in opposite to boolean filters like previous) can have paramete
 
 ## Redirection
 
-Redirection is handled after updating `RouterStore`. Here is the list of properties that should update URL :
- - `action`: get from `routes` the right URL pattern,
- - `mailboxID`: the ID of the mailbox,
- - `MessageID`: the ID of selected message,
- - `query` : filters into messages list,
- - `tab` : change `tabComponent` (ie. `accountEdit`)
+### How works redirection
 
- Whats happening into `routerStore`:
+At first, lets describe routes files:
+ - `client/app/router`: route mapper,
+ - `client/app/stores/router_store`: save status from `RouterActions`,
+ - all other stores are object that manipulate data without storing it.
+ - `client/app/mixins/store_watch_mixin` links `ApplicationComponent` to `Stores:change` events.
 
- Each property related to route is changed after `ActionTypes.ROUTE_CHANGE` into `routerStore` file. After all updates are done, the URL is updated if needed :
 
- ```
- currentURL = @getCurrentURL isServer: false
- if location.hash isnt currentURL
-     _router.navigate currentURL
+### Update application from route
```

Comment:
@m4dz, @aenario : could you give me some feedback aout these explanation, please?